jvector-base/src/main/java/io/github/jbellis/jvector/vector/types/ByteSequence.java

```diff
@@ -31,6 +31,12 @@ public interface ByteSequence<T> extends Accountable
 
     void set(int i, byte value);
 
+    /**
+     * @param byteIndex byte index inside the sequence to start setting short value
+     * @param value short value to set
+     */
+    void setLittleEndianShort(int byteIndex, short value);
```

Comment:
if we're not adding a ShortSequence, maybe indexing shorts in shortIndex instead of bytes would be a minor improvement